command-line/dot-profile.md

```diff
@@ -143,22 +153,9 @@ Add the following to your `.profile` change what your prompt displays: `export P
 Save & open a new Terminal window or `source ~/.profile` to reload this. Notice how the prompt now displays 
 something simliar to `david@Davids-MacBook-Pro` now.
 
-### Independent Practice
-[BASH Programming - Introduction HOW-TO](http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO.html)
-
+(Optional:) More info on - [How to customize your terminal prompt](http://osxdaily.com/2006/12/11/how-to-customize-your-terminal-prompt/)
 
-### Supplemental Material
+Go through the video tutorials mentioned in Material section.
```

Comment:
Please move this up to line 35 so it doesnt look optional.

I think that's it!  Thanks for fixing this up!